pkg/background/generate/generate.go

```diff
@@ -610,6 +643,121 @@ func manageClone(log logr.Logger, apiVersion, kind, namespace, name, policy stri
 	return obj.UnstructuredContent(), Create, nil
 }
 
+func manageCloneList(log logr.Logger, namespace, policy string, clone kyvernov1.Generation, client dclient.Interface) []GenerateResponse {
+	var response []GenerateResponse
+
+	rNamespace := clone.CloneList.Namespace
+	if rNamespace == "" {
+		// return nil, Skip, fmt.Errorf("failed to find source namespace")
+		response = append(response, GenerateResponse{
+			Data:   nil,
+			Action: Skip,
+			Error:  fmt.Errorf("failed to find source namespace"),
```

Comment:
Is `cloneList.Namespace` required? If so, needs to change it here https://github.com/kyverno/kyverno/pull/4384/files#diff-96aa4a3bdb8699fdb9f700c9554e061ebd9578d0c9ea69654a8ecb20d13e3cd8R453.

Reply:
its optional in case of cluster-scope resource, will update it